Updated training style guide #1585 (#1734)
elixir-europe/rdmkit: pages/contribute/page_metadata.md

### More information

> * `training`: List here training material relevant for the page. We recommend to add your training material in TeSS. However, you can also list here training material that is not yet present in TeSS.   The supported registries that can be used in the `registry` attribute are: *YouTube*, *Zenodo*, *Carpentries*, *GitHub* and *TeSS*.

Reviewer: I would argue against YouTube being listed as a registry. Can we distinguish between hosting platforms and registries? I think this would clarify.

Author: I agree with Carpentries. I will remove from examples.
YouTube issue needs to be reviewed by @bedroesb

> * `training`: List here training material relevant for the page. We recommend to add your training material in TeSS. However, you can also list here training material that is not yet present in TeSS.   The supported registries that can be used in the `registry` attribute are: *YouTube*, *Zenodo*, *Carpentries*, *GitHub* and *TeSS*.

Reviewer: I would write the full thing, something like:

We recommend registering your training material in TeSS. Note that TeSS is a registry, and materials can be registered on TeSS independently of the hosting service used. While it is preferable to list here materials on TeSS, we offer a number of additional registry keywords. The full list is...

> * `training`: List here training material relevant for the page. We recommend to add your training material in TeSS. However, you can also list here training material that is not yet present in TeSS.   The supported registries that can be used in the `registry` attribute are: *YouTube*, *Zenodo*, *Carpentries*, *GitHub* and *TeSS*.

Author: i think its too long, people get tired before they even get to the main relevant points. Used gemini for rewording the longer version for ease of reading: 

It is recommended to register your training materials in TeSS, a central registry that enhances the visibility and accessibility of your content. TeSS acts as a directory, meaning your materials can be listed there regardless of where they are hosted.
While TeSS is the preferred platform for registration, a number of other services can be used to host and share your training materials. These include: *YouTube*, *Zenodo*, *GitHub* and *TeSS*.

alternative shorter version: 

To make your training materials more visible, we recommend registering them in TeSS, which is a central registry for training content. You can list your materials on TeSS no matter where they are hosted.
In addition to TeSS, you can also use these platforms for your materials: *YouTube*, *Zenodo*, *GitHub* and *TeSS*.

  For referencing training from a local or national organisation please use this format: 
  
  [Organisation] [Type of material] on [Source]
  
   Examples: 

  * [Organisation] [Training Material/ Search Query] on TeSS
  * [Organisation] on GitHub or [Organisation] - [Repository] in GitHub
  * [Organisation] on YouTube [Channel/ Video]
  * [Organisation] [Training Material/Community] on Zenodo
  * [Organisation] Carpentries Lessons

  For referencing training on a specific topic please use this format:
    
  [Topic]  [Type of material] in [Source] or  [Topic] [Documentation/Website] 
    
  Examples: 

  * [Topic]  [Training material/ Search query] in TeSS
  * [Topic] in GitHub
  * [Topic]  YouTube [channel/video]
  * [Topic] [Training material/Community] in Zenodo
  * [Topic] Carpentries Lessons
 
 ```yml
  training:
    - name: VIB Training material on TeSS 
      registry: TeSS

>       url: https://tess.elixir-europe.org/content_providers/vib-training

Reviewer: This is all the content from a certain provider. I would argue this differs from the categories you mention above (query and materials), as query is a query and URLs for materials look like "https://tess.elixir-europe.org/materials/" + "name_of_content".

Author: VIB is not a query indeed. This is training material by VIB
Query would be ELIXIR Estonia link: https://tess.elixir-europe.org/materials?content_provider=ELIXIR+Estonia

Do you want both examples?

Reviewer: Some examples:

* https://tess.elixir-europe.org/materials?tools=Galaxy
* https://tess.elixir-europe.org/materials?scientific_topics=Genome+annotation&tools=Galaxy

    - name: RStudio basic course in GitHub
      registry: GitHub

>       url: https://github.com/ELIXIREstonia/2025-09-01-R-basic

Author: However we might change wording if we want release info integrated.

 ```

> * `faircookbook`: Here all relevant FAIR Cookbook recipes are listed. This is automatically updated based on the [`faircookbook_rdmkit_mapping.yml`](https://github.com/elixir-europe/faircookbook-rdmkit/blob/main/faircookbook_rdmkit_mapping.yml) mapping file. If you want to make a new link, please make a pull request against this file. Every week the changes of this mapping file are used to update the frontmatter of the corresponding markdown files.

Reviewer: ```suggestion
* `faircookbook`: Here, all relevant FAIR Cookbook recipes are listed. This is automatically updated based on the [`faircookbook_rdmkit_mapping.yml`](https://github.com/elixir-europe/faircookbook-rdmkit/blob/main/faircookbook_rdmkit_mapping.yml) mapping file. To create a new link, please open a pull request against this file. Every week, changes to this mapping file are used to update the frontmatter of the corresponding Markdown files.
```

### More information

* `training`: Add your material directly to TeSS (ELIXIR Training Portal). This ensures maximum visibility and long-term findability. If your material is not yet on TeSS, you can list it manually in the section below using the format url:name.

  For referencing training from a local or national organisation please use this format: 
  
  Templates: 

>   * [Organisation] [Training Material/Search Query] in TeSS

Reviewer: Not necessarily national.

### More information

> * `training`: Add your material directly to TeSS (ELIXIR Training Portal). This ensures maximum visibility and long-term findability. If your material is not yet on TeSS, you can list it manually in the section below using the format url:name.

Reviewer: @diana-pilvar @bedroesb In the text above, it looks like things get treated differently depending on whether something is registered on TeSS or not, but this is not the case, right? All entries on the page follow the same format, independently of whether the URL is a TeSS query or something else. Or have I misunderstood?
